src/extract_wheels.py

```diff
@@ -86,6 +88,10 @@ def extract_wheel(whl, directory, extras):
                 ),
             )
         )
+        f.write(entry_points.entry_point_build_file_targets(
```

Comment:
We should be careful here as this change will leak the prefixed package name as general usage i.e `@pip//pypi__black:black`. Right now we have the requirement macro to avoid needing to know about the `pypi__` prefix.

Reply:
Oh yeh I know. This is WIP. 

Mostly putting this up as PR so we don't step on each other's toes.